test/jasmine/assets/custom_matchers.js

```diff
@@ -71,6 +71,26 @@ module.exports = {
                 };
             }
         };
+    },
+
+    toBeWithin: function() {
+        return {
+            compare: function(actual, expected, tolerance, msgExtra) {
```

Comment:
Nice. I found `toBeCloseTo` a bit unpleasant in the way it uses _digits of precision_ instead of just a tolerance.